### PR TITLE
Added warrens dawn/dusk messages

### PR DESCRIPTION
**Pull request by mvancleave99**
Added a couple of dawn/dusk messages that the game sends when a player is around warrens.

### CODE REVIEW

File: src/clock/mumeclock.cpp [hunk 1]
```diff
@@ -84,6 +84,11 @@ const QHash<QString, MumeTimeEnum> MumeClock::m_stringTimeHash{
     {"The sun slowly climbs from the vale of Anduin in the east.", MumeTimeEnum::DAWN},
     {"Shadows surround you as the sun disappears behind the peaks of the Hithaeglir.",
      MumeTimeEnum::DUSK},
+    
+    // Outside Warrens
+    {"Light gradually filters in, proclaiming a new sunrise.", MumeTimeEnum::DAWN},
+    {"The deepening gloom announces another sunset.", MumeTimeEnum::DUSK},
+    {"The Rising sun illuminates the landscape.", MumeTimeEnum::DAWN},
```
Comment (nschimme) [inline src/clock/mumeclock.cpp:91]: Typo? Capital R in rising?

Comment (mvancleave99) [reply src/clock/mumeclock.cpp:91]: Correct. That is a typo. I'll fix.

Comment (mvancleave99) [reply src/clock/mumeclock.cpp]: Fixed and recommitted.

**Comment by mvancleave99** [comment]
Is there any reason not to use regular expressions for these? That would reduce the number of entries and possibly capture some additional missing regional message.

**Comment by nschimme** [comment]
> Is there any reason not to use regular expressions for these? That would reduce the number of entries and possibly capture some additional missing regional message.

Nobody has coded support for it yet? :-)

I have since added some type of Regex support in this class but I had not get around to combining the two mechanisms: https://github.com/MUME/MMapper/blob/master/src/parser/AbstractParser-Actions.cpp

Please feel free to enhance this yourself!

**Comment by mvancleave99** [comment]
> 

Yeah, I may. That's sort of why I asked. I may need help with getting the C++ part right, but can certainly develop the regex.

**Comment by mvancleave99** [comment]
> 

Nope. Looking at the code you linked, this is way beyond my capability.

**Pull request merged**